DEVELOPMENT.md

```diff
@@ -264,38 +283,66 @@ While iterating on the project, you may need to:
 
 To make changes to these CRDs, you will probably interact with:
 
--   The CRD type definitions in
-    [./pkg/apis/pipeline/v1beta1](./pkg/apis/pipeline/v1beta1)
--   The reconcilers in [./pkg/reconciler](./pkg/reconciler)
--   The clients are in [./pkg/client](./pkg/client) (these are generated by
+- The CRD type definitions in [./pkg/apis/pipeline/v1beta1](./pkg/apis/pipeline/v1beta1)
+- The reconcilers in [./pkg/reconciler](./pkg/reconciler)
+- The clients are in [./pkg/client](./pkg/client) (these are generated by
     `./hack/update-codegen.sh`)
 
-## Install Pipeline
+### Managing Tekton Objects using `ko`
+
+The `ko` command is the preferred method to manage (i.e., create, modifying  or delete) Tekton Objects in Kubernetes from your local fork of the project. Some common operations include:
+
+#### Install Pipeline
 
-You can stand up a version of this controller on-cluster (to your `kubectl
-config current-context`):
+You can stand up a version of this controller on-cluster (to your `kubectl config current-context`):
 
 ```shell
 ko apply -f config/
 ```
 
-### Redeploy controller
+#### Delete Pipeline
+
+You can clean up everything with:
+
+```shell
+ko delete -f config/
+```
+
+#### Redeploy controller
 
 As you make changes to the code, you can redeploy your controller with:
 
 ```shell
 ko apply -f config/controller.yaml
 ```
 
-### Tear it down
+#### Install in custom namespace
 
-You can clean up everything with:
+When changing Tekton Objects it is often helpful to install them into a custom (non-default) namespace to make it easier to manage them as a group and view them in logs.
```

Comment:
I don't totally understand the motivation for this.  What is it about using a namespace other than "tekton-pipelines" that makes things easier to manage and view?

Reply:
> I don't totally understand the motivation for this. What is it about using a namespace other than "tekton-pipelines" that makes things easier to manage and view?

The base text was already in there for changing namespaces (and not 100% confident the existing instructions are complete); would clearly love the canonical use cases as I am reflecting my only perceived value and what I have seen taught in workshops (i.e., using alternate namespaces to run a concurrent development level version and isolate logs via parsing.  I had thought to check "blame" for the inclusion and see if that person could point me to their use cases when they added it.

Reply:
Ah ok, I understand the context a bit more now & hadn't realized the bash script already existed prior to this PR.

It looks like instructions for changing the namespace were added two years ago in cb8de8d39b7f660e9d9d996d2fd183888b476667 without much further commentary in the commit message. You could very well be right that it's to run multiple concurrent development installations.